contracts/exchange/GRVTExchange.sol

```diff
@@ -22,7 +24,13 @@ contract GRVTExchange is
     _disableInitializers();
   }
 
-  function initialize(address admin, address chainSubmitter, address initializeConfigSigner) public initializer {
+  function initialize(
+    address admin,
+    address chainSubmitter,
+    address initializeConfigSigner,
+    address beaconOwner,
+    bytes32 depositProxyProxyBytecodeHash
```

Comment:
is `ProxyProxy` name intentional?

Reply:
Yes, it's the `BeaconProxy` for the `DepositProxy`.